lib/ansible/module_utils/common/validation.py

```diff
@@ -521,7 +521,9 @@ def check_type_bytes(value):
 
 
 def check_type_bits(value):
-    """Convert a human-readable string value to bits
+    """Convert a human-readable string bits value to bits in integer
```

Comment:
Since you already improve docstrings, it'd be nice to follow PEP 257 and put periods at the end of docstring title phrases.

Reply:
yep, I should make a check list for writing tests! going to reread all unit tests and make it.